### DISCUSSION TITLE
what time can support long running transactions

### DISCUSSION THREAD

**Discussion by AoiIsMine** [prisma/prisma#7409] [Q&A] [answered]
Dear Author:
    I want to use  long running transactions,
but it's not supported yet, so I would like to ask when it will be supported,
will version 2.25 support this feature
or it's no longer planned to develop
    Thank you for your answer

**Comment by ryands17** [comment, marked as answer]
@AoiIsMine 👋 

Have a look at this thread: https://github.com/prisma/prisma/issues/1844#issuecomment-846477520

**Reply by AoiIsMine** [reply]
Thank you for your reply. I've read it before and missed the last words
It looks like we'll have to wait a few months
I can only do it in other ways
Look forward to coming earlier

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
